modules/bookshelf.js

```diff
@@ -0,0 +1,38 @@
+export default class BookShelf {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('book')) || [{ title: 'title 1', author: 'author 2' }, { title: 'title 1', author: 'author 2' }];
+  }
+
+  add(title, author) {
+    this.books.push({ title, author });
+    localStorage.setItem('book', JSON.stringify(this.books));
+  }
+
+  remove(book) {
+    this.books = this.books.filter((b) => b !== book);
+    localStorage.setItem('book', JSON.stringify(this.books));
+  }
+
+  render() {
+    const bookList = document.querySelector('#book-list');
+    bookList.innerHTML = '';
+    for (let i = 0; i < this.books.length; i += 1) {
+      const book = this.books[i];
+      const li = document.createElement('li');
+      li.innerHTML = `${book.title} by ${book.author}`;
+      const deleteButton = document.createElement('button');
+      deleteButton.textContent = 'Remove';
+      deleteButton.classList.add('deleteButton');
+      deleteButton.addEventListener('click', () => {
+        this.remove(book);
+        this.render();
+      });
+      li.appendChild(deleteButton);
+      bookList.appendChild(li);
+
+      if (i % 2 === 0) {
+        li.style.backgroundColor = '#e9e9e9';
+      }
+    }
```

Comment:
[OPTIONAL] The work is awesome 👍🏽 like the books 😆 but in order to further improve it, I'd suggest adding a check to show another message if there are no books in the list. Thoughts? 🤔 

This is what you have so far 👇🏽 

<img width="969" alt="Screenshot 2023-03-27 at 07 57 51" src="https://user-images.githubusercontent.com/45739963/227935155-7f75f1a5-f1cc-4945-8d76-084d0b9040ac.png">